### PR TITLE
Use correct url protocol for show view of images

### PR DESCRIPTION
**Pull request by donni106**
- if the website setting is set to use ssl, the image urls should be presented with
  the correct https:// protocol
- until now it was hardcoded http:// and users copied and used it also in
  https:// pages which result in website warnings and/or errors

### CODE REVIEW

File: app/assets/javascripts/goldencobra/html_editors/ckeditor/config.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ CKEDITOR.editorConfig = function(config) {
   // The toolbar groups arrangement, optimized for two toolbar rows.
   config.toolbarGroups = [
     { name: 'clipboard',   groups: [ 'clipboard', 'undo' ] },
-    { name: 'editing',     groups: [ 'find', 'selection', 'spellchecker' ] },
```
Comment (donni106) [inline app/assets/javascripts/goldencobra/html_editors/ckeditor/config.js]: Did not change this in the current work and did not remember, but it seems correct. No spellchecker wanted in ckeditor, a minimal setup is preferred.

File: Gemfile.lock [hunk 2]
```diff
@@ -98,7 +98,7 @@ GEM
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
     active_model_serializers (0.9.5)
       activemodel (>= 3.2)
-    activeadmin (1.0.0.pre4)
+    activeadmin (1.0.0.pre2)
```
Comment (5minpause) [inline Gemfile.lock]: Why did you downgrade these different gems here?

File: admin/uploads.rb [hunk 3]
```diff
@@ -88,25 +91,45 @@
 	  end
     column "" do |upload|
       result = ""
-      result += link_to(t(:view), admin_upload_path(upload), class: "member_link edit_link view", title: I18n.t("active_admin.uploads.title1"))
-      result += link_to(t(:edit), edit_admin_upload_path(upload), class: "member_link edit_link edit", title: I18n.t("active_admin.uploads.title2"))
-      result += link_to(t(:delete), admin_upload_path(upload), method: :DELETE, "data-confirm" => t("delete_article", scope: [:goldencobra, :flash_notice]), class: "member_link delete_link delete", title: I18n.t("active_admin.uploads.title3"))
+      result += link_to(
```
Comment (5minpause) [inline admin/uploads.rb]: Your changes to incorporate our style guide are well received. Next time please put them into their own commit. It makes finding the actual change of the PR quite difficult.

File: admin/uploads.rb [hunk 4]
```diff
@@ -88,25 +91,45 @@
...
+        "data-confirm" => t("delete_article", scope: [:goldencobra, :flash_notice]),
+        class: "member_link delete_link delete",
+        title: I18n.t("active_admin.uploads.title3")
+      )
       raw(result)
     end
   end
 
   show do
+    protocol = Goldencobra::Setting.for_key("goldencobra.use_ssl") == "true" ? "https" : "http"
```
Comment (5minpause) [inline admin/uploads.rb]: Searching across Goldencobra for the occurences of `Goldencobra::Setting.for_key("goldencobra.use_ssl")` reveals at least 5 hits for the current master branch. Adding a 6th here doesn't help the cause of bettering Goldencobra.

We should strive for a solution where building an url is encapsulated somewhere else. It always comes down to building a url from the settings of `Goldencobra::Setting.for_key("goldencobra.use_ssl")` and `Goldencobra::Setting.for_key("goldencobra.url")`. I don't like particularly, that you build the url anew for every row in the table.

File: Gemfile.lock (hunk 2 above)
Comment (donni106) [inline Gemfile.lock]: Needed to downgrade to make `bundle update` work. Researches brought me to:
https://github.com/activeadmin/activeadmin/issues/3926#issuecomment-233651735

File: admin/uploads.rb (hunk 4 above)
Comment (donni106) [inline admin/uploads.rb]: Yes, you are right.

**Comment by 5minpause** [comment]
My pleasure
